src/app/mat-main-nav/mat-main-nav.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "@angular/material/core/style/_variables";
+
+.mat-toolbar-row,
+.mat-toolbar-single-row {
+  @media screen and (#{$mat-xsmall}) {
+    height: 72px;
+  }
+}
+
+.mat-icon-button.logo-icon {
+  width: 100px;
+  height: 40px;
+  line-height: 1;
+}
+
+.sidenav-container {
+  height: 100%;
+}
+
+.sidenav {
+  width: 90%;
+}
+
+.icon_nhs_logo.mat-icon {
+  width: 100px;
+  height: 40px;
+
+  &:hover {
+    box-shadow: 0 0 0 4px #003d78;
+  }
+}
+
+.mat-toolbar.mat-primary {
+  position: sticky;
+  top: 0;
+  z-index: 1;
+}
+
+.fill-space {
+  flex: 1 1 auto;
+  padding-left: 16px;
+}
+
+.mat-icon-button,
+.mat-drawer {
+  &.hidden {
+    display: none;
+  }
+}
+
+// TO DO extract all to mixins for re-use with media queries
+.no-decoration-link {
```

Comment:
this class as well as the other 2 below could benefit from living inside a utlilities or some sort of shared scss file

Reply:
done